dusty/compiler/spec_assembler.py

```diff
@@ -149,3 +149,16 @@ def get_all_repos(active_only=False, include_specs_repo=True):
         if spec['repo']:
             repos.add(Repo(spec['repo']))
     return repos
+
+def _get_dependent_repos_for_app_or_library(app_or_library_name):
+    specs = get_expanded_libs_specs()
+    spec = specs.get_app_or_lib(app_or_library_name)
+    repos = set()
+    for dependent_name in spec['depends']['libs']:
+        repos.add(Repo(specs.get_app_or_lib(dependent_name)['repo']))
+    return repos
+
+def get_all_repos_for_app_or_library(app_or_library_name):
```

Comment:
True.  However since libraries don't have containers on their own, it's still slightly off, no?  They have their own containers for testing purposes (again testing-specific)